apps/gp2-server/src/handlers/utils.ts

```diff
@@ -0,0 +1,36 @@
+import { AlgoliaClient, gp2, SavePayload } from '@asap-hub/algolia';
+import { ListResponse } from '@asap-hub/model';
+import logger from '../utils/logger';
+
+type PayloadType<T> = T extends gp2.Payload ? T['type'] : never;
+
+export const createProcessingFunction =
+  <T extends gp2.Payload, Type extends PayloadType<T>>(
+    algoliaClient: AlgoliaClient<'gp2'>,
+    type: Type,
+  ) =>
+  async (found: ListResponse<T['data']>) => {
+    logger.info(
+      `Found ${found.total} items. Processing ${found.items.length} items.`,
```

Comment:
really minor, but maybe we could have 

```suggestion
      `Found ${found.total} items. Processing ${found.items.length} ${type}s.`,
```

and the same thing where we have the other logger info at line 35